src/hotspot/os_cpu/linux_arm/os_linux_arm.cpp

```diff
@@ -209,8 +209,10 @@ frame os::fetch_compiled_frame_from_context(const void* ucVoid) {
 }
 
 intptr_t* os::fetch_bcp_from_context(const void* ucVoid) {
-  Unimplemented();
-  return nullptr;
+  assert(ucVoid != nullptr, "invariant");
+  const ucontext_t* uc = (const ucontext_t*)ucVoid;
+  assert(os::Posix::ucontext_is_interpreter(uc), "invariant");
+  return (intptr_t*)uc->uc_mcontext.arm_r7;
```

Comment:
You sure it is always R7? The `Rbcp` register definition seems to choose between R7 and R11: https://github.com/openjdk/jdk/blob/241808e13fb032b0ec192e0b7ff94891a653ac94/src/hotspot/cpu/arm/register_arm.hpp#L414-L415